browsers/opera.json

```diff
@@ -307,22 +307,28 @@
         "53": {
           "release_date": "2018-05-10",
           "release_notes": "https://dev.opera.com/blog/opera-53/",
-          "status": "current"
+          "status": "retired"
         },
         "54": {
-          "status": "beta"
+          "release_date": "2018-06-28",
+          "release_notes": "https://dev.opera.com/blog/opera-54/",
+          "status": "retired"
         },
         "55": {
-          "status": "nightly"
+          "release_date": "2018-08-16",
+          "release_notes": "https://blogs.opera.com/desktop/2018/09/opera-55-0-2994-61-stable-update/",
```

Comment:
I think https://blogs.opera.com/desktop/2018/08/opera-55-offers-better-control-web-pages-accessible-bookmarks/ would be better.